python_modules/libraries/dagster-duckdb/dagster_duckdb/io_manager.py

```diff
@@ -106,6 +105,30 @@ def duckdb_io_manager(init_context):
     return duckdb_io_manager
 
 
+class ConfigurableDuckDBIOManager(ConfigurableIOManagerFactory):
+    database: str
+    schema_: Optional[str] = Field(None, alias="schema")  # schema is a reserved word for pydantic
+
+    @staticmethod
+    @abstractmethod
+    def type_handlers() -> Sequence[DbTypeHandler]:
+        ...
+
+    @staticmethod
+    def default_load_type() -> Optional[Type]:
```

Comment:
This will continue to follow the defaulting logic if the user doesn't override it (pulling from the type handler if there's only one), right?

Reply:
yeah, the default return of None will make the DB IO manager pull from the type handler